quesma/quesma/config/config_v2.go

```diff
@@ -709,14 +833,34 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 			}
 
 			processedConfig.IngestTarget = make([]string, 0) // reset previously set defaultConfig.IngestTarget
-			for _, target := range indexConfig.Target {
-				if targetType, found := c.getTargetType(target); found {
-					processedConfig.IngestTarget = append(processedConfig.IngestTarget, targetType)
-				} else {
-					errAcc = multierror.Append(errAcc, fmt.Errorf("invalid target %s in configuration of index %s", target, indexName))
+			targets, errTarget = c.getTargetsExtendedConfig(indexConfig.Target)
```

Comment:
This `if len(targets) > 0`, `if len(targets) == 0` pattern repeats a few times in the code. Maybe this logic should be moved into `c.getTargetsExtendedConfig`?

What I mean by that is that if simplified target configuration is used, then `getTargetsExtendedConfig` would "translate" simplified target configuration into extended configuration: translate `[target1, target2]` into array of structs `[{target: target1, properties: {}}, {target: target2, properties: {}}]`

Reply:
Yes, we can do that, but as I mentioned. I would prefer to do that in next PR if you don't mind, to have working version first